start-site/src/main/resources/application.yml

```diff
@@ -782,7 +782,7 @@ initializr:
             - compatibilityRange: "[2.2.0.M1,2.3.0.M1)"
               version: 3.3.0
             - compatibilityRange: "[2.3.0.M1,2.4.0-M1)"
-              version: 3.4.3
+              version: 3.5.0
```

Comment:
`2.3.x` is in maintenance phase and Spring Boot 2.4 will be released in a month or so. Would it be worth mapping this version with Spring Boot 2.4 rather than bumping to a new feature release now?

Reply:
Camel 3.6 will upgrade to SB 2.4 and is planned for 6-8 weeks or so.

Reply:
Great news, thanks for the feedback @davsclaus